packages/apps/marmalade-marketplace/src/components/MintToken/index.tsx

```diff
@@ -92,39 +126,63 @@ function MintTokenComponent() {
+                alt="Token Image"
+                className={styles.tokenImageClass}
+              />
+            </div>
+            <div className={styles.formContainer} >
+              <TextField label="Token ID" name="tokenId" value={tokenId} onChange={handleTokenInputChange} onBlur={fetchTokenInfo} /> 
+              <NumberField label="Amount" value={amount} onValueChange={hanldeAmountInputChange} />
+            </div>
+          </CrudCard>
+
+          {result && (<CrudCard
+            title="Token Policy Information"
             description={[
-              "Mint a new token",
-              "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi",
-              "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore",
-              "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia"
-            ]}
-          >
-          <TextField label="Token ID" name="tokenId" value={tokenId} onChange={handleTokenInputChange} />
-          <NumberField label="Amount" value={amount} onValueChange={hanldeAmountInputChange} />
-
-        </CrudCard>
-
-        <div className={styles.buttonRow}>
-          <Button type="submit" onClick={handleButtonClick}>
-            Mint Token
-          </Button>
-        </div>
-        {result && (<CrudCard
-          title="Token Policy Information"
-          description={[
-            "Displays the token policy information",
-          ]}>
-            <p>Token Policy Information: {JSON.stringify(result)}</p>
-        </CrudCard>
-        )}
+              "Displays the token policy information",
+            ]}>
+            <div className={styles.checkboxRow}>
+              <Checkbox isReadOnly={true} id="nonUpdatableURI" isSelected={result.nonUpdatableURI}>Non-Updatable URI</Checkbox>
+              <Checkbox isReadOnly={true} id="guarded"  isSelected={result?.guarded}>Guarded</Checkbox>
+            </div>
+            <div className={styles.checkboxRow}>
+              <Checkbox isReadOnly={true} id="nonFungible" isSelected={result?.nonFungible}>Non Fungible</Checkbox>
+              <Checkbox isReadOnly={true} id="hasRoyalty" isSelected={result?.hasRoyalty}>Has Royalty</Checkbox>
+            </div>
+            <div className={styles.checkboxRow}>
+              <Checkbox isReadOnly={true} id="collection" isSelected={result?.collection}>Collection</Checkbox>
+            </div> 
+          </CrudCard>
+          )}
       </Stack>
+      <div className={styles.buttonRow}>
+        <Button variant="outlined" onPress={onCancelPress}>
+          Cancel
+        </Button>
+        <Button type="submit" onClick={handleButtonClick}>
+          Mint Token
+        </Button>
+      </div>
+      </div>
       ) : (
-        <SendTransaction send={send} preview={preview} poll={poll} transaction={transaction} />
+        <SendTransaction send={send} preview={preview} poll={poll} transaction={transaction}  />
```

Comment:
Since we're now redirecting to the transaction page we don't need the `SendTransaction` component on this page anymore.